crates/rust-analyzer/src/config.rs

```diff
@@ -150,6 +150,10 @@ config_data! {
         ///
         /// Set to `"all"` to pass `--all-features` to Cargo.
         check_features | checkOnSave_features: Option<CargoFeaturesDef>  = "null",
+        /// List of check diagnostics to ignore.
```

Comment:
Can you clarify it a bit more? For example: List of `cargo check` (or other command specified in `check.command`) diagnostics to ignore.

Reply:
Gentle ping @adamse

Reply:
thank you. i’ll update in a week or so!